### ISSUE TITLE
Safari currentTime display wrong

### ISSUE THREAD

**Issue by seaside98** [videojs/video.js#3672]
## Description
Safari 10.0
MacOS Sierra 10.12
Videojs 5.11.7 and 5.12
When clicking to scrub a paused video in safari 10, the currentTime display remains one behind the actual position. For example, if I click to scrub to 10 seconds, it still displays 0:00. Then if I scrub to 50 seconds, it displays 0:10, and so on.

[JS Bin](http://jsbin.com/gixepezahi/edit?html,css,output)

## Steps to reproduce

1. Begin playing the video, and then pause it.
2. Click the slider to scrub to a point in the video. Repeat this a couple of times.
3. The currentTime will be one click delayed.

This bug appears to be limited to Safari 10.0.

**Comment by misteroneill** [comment]
This sounds like the issue that is fixed by v5.11.8 - please re-open if it's not!

**Comment by seaside98** [comment]
@misteroneill Still broken in v5.11.8.

**Comment by hartman** [comment]
I confirm @seaside98 's report. It sometimes seems to work, but the actual currentTime displayed is often incorrect.

**Comment by gkatsev** [comment]
I can confirm. That's bizarre. I get a different, almost random, value each time I click to seek.

**Comment by gkatsev** [comment]
Actually, I can confirm it in the jsbin above but that's using videojs 5.0, when I updated it to 5.11, I'm not seeing any issues.

**Comment by seaside98** [comment]
![out](https://cloud.githubusercontent.com/assets/6372893/19528427/97611c1c-95f1-11e6-9440-8326f2d48309.gif)
I'm still seeing it with 5.11. Are you using Safari 10.0?

**Comment by gkatsev** [comment]
Yeah, trying out on safari 10. I'll investigate some more, I guess but it seems to be working fine for me.

**Comment by gkatsev** [comment]
Unfortunately, I'm still unable to reproduce this.

**Comment by marguinbc** [comment]
@gkatsev I can still repro with 5.12 or 5.13.1
play
pause
click on progress bar to seek
almost appears that the previous current time sticks to the tooltip

**Comment by misteroneill** [comment]
Could it be the fact that it's using a cached `currentTime` if it thinks the player is scrubbing?

**Comment by boushley** [comment]
This reproduces for me in Safari 10.0.1 using video.js 5.13.1

It does not reproduce in Chrome 55 for me.

**Comment by boushley** [comment]
This also reproduces in Safari Tech Preview (10.1)

**Comment by boushley** [comment]
In some further testing this appears to be related to the fact that seeking appears instant on this video. If I try a different video like this one: https://download.blender.org/demo/movies/tears-of-steel_teaser.mp4 seeking is no longer instant. With seeking taking longer this bug no longer shows up in Safari.

The behavior when seeking takes some time is:

1) Original time is displayed
2) Click seek
3) Seeking spinner starts - time continues to display original time.
4) Seeking completes - time display updates to show time that we seeked to.

The fact that step 3 continues to show the original time seems a bit odd since as a user I expect the value to update to what I just saw as I as hovering, but I can understand why our code waits until the seek completes.

My guess is that a seek end event is being missed in this repro js bin.

**Comment by boushley** [comment]
@misteroneill do you happen to have a link to the issue from `v5.11.8` that was similar to this?

**Comment by misteroneill** [comment]
@boushley It was #3349
